### ISSUE TITLE
Windows 10 Version 1803: support hardware keyboard suggestion announcement

### ISSUE THREAD

**Issue by josephsl** [nvaccess/nvda#8189]
Hi,

Related to #7273: in Version 1803, it is now possible to let people see suggestions when they type on hardware keyboards 0United States English). At the moment NVDA does not announce this, but add a possibility to do so.

This will be part of #7273 due to same app module in use.

Thanks.

**Comment by josephsl** [comment]
Hi,

on second thought, it might be best to do it after #7273 pull request hits the air, because hardware suggestions handling requires careful coordination. Another approach is to treat this as suggestions event.

Thanks.

**Comment by Adriani90** [comment]
Hey @josephsl this should be solved now meanwhile in Windows 10 with NVDA 2019.1.1. Or am I wrong?

**Comment by josephsl** [comment]
Hi, resolved in 2018.  Thanks, go ahead and close this.

 ----- Original Message -----

**Comment by Adriani90** [comment]
Thanks Joseph! Closing as works for me.
